qa/standalone/test_ceph_daemon.sh

```diff
@@ -153,7 +156,25 @@ $SUDO $CEPH_DAEMON shell --fsid $FSID --config $CONFIG --keyring $KEYRING -- \
 # WRITE ME
 
 ## adopt
-# WRITE ME
+tar xzvf $DATA_DIR_TAR -C $TMPDIR
+for t in mon mgr mds; do
+	$SUDO $CEPH_DAEMON --data-dir ${TMPDIR}/var/lib/ceph \
+	                   --unit-dir ${TMPDIR}/etc/systemd/system \
```

Comment:
I'm a bit nervous about this approach.  The rest of these tests do the test in the default locations, with a bogus fsid.  But here, we're doing everything in a temp dirs: that's both the source (legacy) and target (new).  I wonder if, instead, we should have a separate set of options specific to adopt that is the data_dir for the legacy daemons, but the adopt them into our normal locations.  This would remove the need for the last 2 patches in #31487, I think.

Reply:
Agree, this has been refactored in #31487 to use a `--base-dir` instead of intermixing the src and dst dirs.